### DISCUSSION TITLE
Render file type restriction in the input accept attribute

### DISCUSSION THREAD

**Discussion by thiagorossener** [solspace/craft-express-forms#67] [Feature Requests]
### Description

Hi, I couldn't find in the documentation, is there a way to do that?

Basically, I'd like to show this:

![image](https://user-images.githubusercontent.com/1750826/86532686-3f0cbc00-bea2-11ea-9c84-9b4b9ec400da.png)

Like this:

```
<input type="file" id="cv" name="cv" accept=".doc,.docx,.pdf">
```

Thank you.

**Comment by kjmartens** [comment]
@thiagorossener Sorry for the delay. There currently isn't a way to do this with Express Forms. It's not something on our roadmap for now, but I've added it to the feature requests list though. :)

**Comment by thiagorossener** [comment]
No worries @kjmartens, thanks for the response.
